sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -1914,9 +1914,12 @@ object SQLConf {
     .createOptional
 
   val V2_SESSION_CATALOG = buildConf("spark.sql.catalog.session")
-      .doc("Name of the default v2 catalog, used when a catalog is not identified in queries")
+      .doc("A catalog implementation which will be used as the session catalog for Spark " +
```

Comment:
After the last sync discussion, I'm not sure that "will be used as the session catalog" is clear. I like the wording you used in the description of `CatalogExtension` when you referred to "Spark's built-in session catalog". How about changing this to "a catalog implementation that will be used in place of the Spark built-in session catalog for v2 operations".